### PR TITLE
refactor: Cleaned up the codebase

### PR DESCRIPTION
**Pull request by BlazeMCworld**
- Formatted code
- Updated deprecated code
- Removed unused imports

### CODE REVIEW

**Comment by Chickensoup20** [comment]
😍

File: src/main/java/hypersquare/hypersquare/command/DeleteAllPlotsCommand.java [hunk 1]
```diff
@@ -23,17 +25,18 @@
...
-            Player player = (Player) sender;
+        if (sender instanceof Player player) {
             if (player.hasPermission("hypersquare.dumplots")) {
-                Utilities.sendInfo(player,"Dumped all plots");
-                for (Player player1 : Bukkit.getOnlinePlayers()){
-                    player1.kickPlayer(ChatColor.translateAlternateColorCodes('&',"&c&lKick from Hypersquare \n &fReason: massive dump"));
+                Utilities.sendInfo(player, "Dumped all plots");
+                for (Player player1 : Bukkit.getOnlinePlayers()) {
+                    player1.kick(Component.text("Kick from Hypersquare").color(NamedTextColor.RED).decoration(TextDecoration.BOLD, true)
+                            .append(Component.text("Reason: massive dump").color(NamedTextColor.RED))
```
Comment (RedVortexDev) [inline src/main/java/hypersquare/hypersquare/command/DeleteAllPlotsCommand.java:33]: Not the same as a \n

File: src/main/java/hypersquare/hypersquare/command/LocateCommand.java [hunk 2]
```diff
@@ -27,8 +28,12 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
             } else {
                 target = player;
             }
+            if (target == null) {
+                player.sendMessage(Component.text("Player not found.").color(NamedTextColor.RED));
```
Comment (RedVortexDev) [inline src/main/java/hypersquare/hypersquare/command/LocateCommand.java]: Should be replaced with Utilities.sendError()

File: src/main/java/hypersquare/hypersquare/plot/CodeBlockManagement.java [hunk 3]
```diff
@@ -49,62 +41,39 @@ public static void placeBracket(Location location, String codeblock, String brac
...
-        return location;
-    }
-
-    public static Location findCodelineStartLoc(Location location){
-        location.setZ(1);
-        return location;
-    }
-
-    public static Location findNextBracket(Location location){
-        while (location.clone().add(0,0,1).getBlock().getType() != Material.AIR || location.clone().add(0,0,2).getBlock().getType() != Material.AIR || location.clone().getBlock().getType() != Material.AIR){
```
Comment (RedVortexDev) [inline src/main/java/hypersquare/hypersquare/plot/CodeBlockManagement.java]: why were this deleted?

**Review by RedVortexDev — CHANGES_REQUESTED** [review]

**Review by RedVortexDev — APPROVED** [review]
LGTM
